src/services/user.js

```diff
@@ -517,9 +517,9 @@ const getAdvisors = async (username) => {
 };
 
 const getCliftonStrengths = async (username) => {
-  try {
-    let cliftonStrengths = await http.get(`profiles/clifton/${username}/`);
+  let cliftonStrengths = await http.get(`profiles/clifton/${username}/`);
 
+  if (cliftonStrengths.Strengths !== null && cliftonStrengths.Strengths[0] !== null) {
```

Comment:
You could simplify this check by using [`optional chaining`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and checking for a ['truthy' value](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) instead of just `null`:
```suggestion
  if (cliftonStrengths?.Strengths?.[0]) {
```